itou/templates/apply/includes/siae_actions_modals.html

```diff
@@ -0,0 +1,46 @@
+{% load matomo %}
+{% load str_filters %}
+
+{% if batch_mode %}
+    <div id="batch-action-modals" {% if request.htmx %}hx-swap-oob="true"{% endif %}>
+        {% if can_archive %}
+            <div id="archive_confirmation_modal" class="modal" tabindex="-1" aria-labelledby="archive_modal_title" aria-hidden="true">
+                <div class="modal-dialog modal-dialog-centered">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h2 class="modal-title" id="archive_modal_title">
+                                Archiver {{ selected_nb }} candidature{{ selected_nb|pluralizefr }}
+                            </h2>
+                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Fermer"></button>
+                        </div>
+                        <div class="modal-body">
+                            <p>
+                                Êtes-vous sûr de vouloir archiver
+                                {% if selected_nb > 1 %}
```

Comment:
Idée fumeuse de fin de journée :
```suggestion
                                {% if selected_nb|pluralizefr %}
```

Reply:
Ça marche, ça permettrait de centraliser la règle de pluriel à un seul endroit mais ça n'aide pas la relecture :sweat_smile: donc je vais plutôt garder la version KISS :)

Reply:
En retombant sur la même logique mais dans une vue cette fois je me suis dit qu'un avantage c'est que ça donne du contexte au `if`, on sais que c'est pour de l'affichage et pas de la logique métier, et donc à tester un jour pour voir ce que ça donne.